components/profile_popover/profile_popover.jsx

```diff
@@ -345,10 +372,23 @@ class ProfilePopover extends React.Component {
             />
         );
 
+        const supportsChannelAdmin = this.props.isLicensed;
+        const isChannelAdmin = supportsChannelAdmin && (Utils.isChannelAdmin(this.props.isLicensed, this.props.channelMember.roles) || this.props.channelMember.scheme_admin);
+
+        let roleTitle;
+        if (Utils.isSystemAdmin(this.props.user.roles)) {
+            roleTitle = <span className='user-popover__role'>{'System Admin'}</span>;
+        } else if (Utils.isInRole(this.props.teamMember.roles, 'team_admin')) {
+            roleTitle = <span className='user-popover__role'>{'Team Admin'}</span>;
```

Comment:
This should just be `this.props.teamMember.scheme_admin` as with the channel admin case.